### ISSUE TITLE
Maestral can't start due to upstream changes in python-keyring

### ISSUE THREAD

**Issue by GuessWhatBBQ** [samschott/maestral#723]
**Describe the bug**

Possible regression caused by upstream changes to python-keyring, which python-keyring-alt depends on. Tracked by
[this issue](https://github.com/jaraco/keyrings.alt/issues/47). Hopefully should be a speedy fix as there's already a PR addressing it.

**To Reproduce**

simply run `maestral start`

**Expected behaviour**

Error free start 

**System:**

 - Maestral version: 1.6.3
 - Python version: 3.10.6
 - OS: EndeavorOS
 - Desktop environment: i3
 - PyQt version (for Linux GUI): 6.3.1

**Additional context**

Relevant error from the python traceback
```
ImportError: cannot import name 'properties' from 'keyring.util' (/usr/lib/python3.10/site-packages/keyring/util/__init__.py)
```

**Comment by ret2src** [comment]
Quick and dirty solution until [`python-keyrings-alt`](https://archlinux.org/packages/community/any/python-keyrings-alt/) (or your distro equivalent) gets updated to `4.2.0`:

``` bash
sudo pip install keyrings.alt --upgrade
```

Tested on Arch.

**Comment by GuessWhatBBQ** [comment]
Issue seems to be resolved after updating to python-keyring 23.9.3-1

**Comment by samschott** [comment]
Excellent, thanks for the update!
